### ISSUE TITLE
Support metadata removal for Office documents

### ISSUE THREAD

**Issue by ghost** [szTheory/exifcleaner#162]
How about clearing metadata of Office Documents made by Apps such as Microsoft Office, Apache OpenOffice, OnlyOffice and Libre Office ?

**Comment by szTheory** [comment]
Right now we only support the same file types as the underlying `exiftool`, but I would still be open to adding this. Either by finding another library in JavaScript or another language that takes care of it, or writing it from scratch.

**Comment by Loki-L1130** [comment]
I think, You can take a look on MAT2. https://0xacab.org/jvoisin/mat2/-/releases/0.12.1 

Lot of implementation can also be done from there.

**Comment by Loki-L1130** [comment]
But the Problem is, It is Python3.

**Comment by szTheory** [comment]
Oh yeah integrating that as well could be a pain, though still possible. Perhaps using an version of Python embedded in the runtime, or shipping Python along ExifCleaner, or just requiring it as a dep. Could also try and port the code specific to Office to Perl or JS. Or see about contributing directly to exiftool to get support added there.

On the other hand I noticed someone made a GUI app based on MAT2. It appears to be Linux only, but looks pretty sweet! https://metadatacleaner.romainvigier.fr/
![mc1](https://user-images.githubusercontent.com/28652/155517954-5ce1e463-08cf-4c88-ae0c-84eeed6b4f2c.png)
![mc2](https://user-images.githubusercontent.com/28652/155517959-0f122215-489f-4759-ab83-77e267f50c4c.png)
![mc3](https://user-images.githubusercontent.com/28652/155517969-c6765456-f707-4347-a06d-828e70b072b9.png)
![mc4](https://user-images.githubusercontent.com/28652/155517987-42d671c6-4351-4f43-a7dc-e34d47303f86.png)

**Comment by Loki-L1130** [comment]
Yeah, I know that. The Main thing with MAT2 cli is it requires a lot of dependencies to run. It is hard in Windows to Use while as it is easy for MacOS as there are official Instructions. While Linux users are lucky that they have a GUI App.

Porting that code would also be great. But, Contributing to Exiftool would also be great.

They do also support .mkv files. May be that could also be taken into account.

**Comment by szTheory** [comment]
Closing as out of scope.

ExifCleaner's accepted formats are an explicit list (`src/domain/files/file_types.ts:4-37`): images, a set of video/audio containers, and PDF. Office documents aren't on it, and adding them isn't a matter of flipping a switch. An OOXML file is a ZIP of XML parts, with metadata spread across `docProps/core.xml`, `docProps/app.xml`, revision identifiers, and frequently tracked-change and comment history. That's a document-rewriting problem, not a metadata-tag problem, and the bundled ExifTool cannot write those files safely — the same reason DOCX is explicitly refused rather than silently passed through (see #339).

Doing it properly would mean a separate OOXML implementation carrying its own correctness and corruption risk, on files people care about a great deal. I'd rather refuse the format clearly than half-clean someone's contract and have them believe it was scrubbed.
